stdlib/encoding/src/main/java/org/ballerinalang/stdlib/encoding/nativeimpl/Decode.java

```diff
@@ -37,15 +39,16 @@ public static Object decodeBase64Url(String input) {
             byte[] output = Base64.getUrlDecoder().decode(input);
             return BValueCreator.createArrayValue(output);
         } catch (IllegalArgumentException e) {
-            return EncodingUtil.createError("Input is not a valid Base64 URL encoded value");
+            return EncodingUtil.createError("Input is not a valid Base64 URL encoded value", DECODING_ERROR);
```

Comment:
@Bhashinee I think we need to define this distinct error DECODING_ERROR. the only it will get generated properly at ballerina level with typeIds. 

It's is required that the typeId to be same name as the defined type for distinct types.
For example type A distinct error, here typeId of A is A.